transport-classes-io_uring/src/main/java/io/netty/incubator/channel/uring/AbstractIOUringStreamChannel.java

```diff
@@ -211,12 +211,13 @@ protected Executor prepareToClose() {
 
         private ByteBuf readBuffer;
         private IovArray iovArray;
+        private final ByteBuf iovArrayBuffer = alloc().directBuffer(Limits.IOV_MAX * IovArray.IOV_SIZE);
```

Comment:
This allocates a huge amount of direct memory that will stay alive the whole life time of the connection. This is too wasteful.